test/val/bug1310.c

```diff
@@ -68,6 +68,21 @@ int main (void)
         failures++;
     }
 
+    if (-32768U != 32768U) {
+        fprintf (stderr, "Expected -32768U == 32768U, got: %ld\n", (long)-32768U);
+        failures++;
+    }
+    if (~32767U != 32768U) {
+        fprintf (stderr, "Expected ~32767U == 32768U, got: %ld\n", (long)~32767U);
+        failures++;
+    }
     printf ("failures: %u\n", failures);
```

Comment:
A `printf()` is below the next test.  Therefore, this line must be removed (it's redundant).

Reply:
Fixed.